File: cSharp/Api-ng-sample-code/Api-ng-sample-code/TO/Order.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Newtonsoft.Json;

namespace Api_ng_sample_code.TO
{
    public class Order
    {
        [JsonProperty(PropertyName = "betId")]
        public string BetId { get; set; }

        [JsonProperty(PropertyName = "orderType")]
        public OrderType OrderType { get; set; }

        [JsonProperty(PropertyName = "status")]
        public OrderStatus Status { get; set; }

        [JsonProperty(PropertyName = "persistenceType")]
        public PersistenceType PersistenceType { get; set; }

        [JsonProperty(PropertyName = "side")]
        public Side Side { get; set; }

        [JsonProperty(PropertyName = "price")]
        public double Price { get; set; }

        [JsonProperty(PropertyName = "size")]
        public double Size { get; set; }

        [JsonProperty(PropertyName = "bspLiability")]
        public double? BspLiability { get; set; }

        [JsonProperty(PropertyName = "placedDate")]
        public DateTime? PlacedDate { get; set; }

        [JsonProperty(PropertyName = "avgPriceMatched")]
        public double? AvgPriceMatched { get; set; }

        [JsonProperty(PropertyName = "sizeMatched")]
        public double? SizeMatched { get; set; }

        [JsonProperty(PropertyName = "sizeRemaining")]
        public double? SizeRemaining { get; set; }

        [JsonProperty(PropertyName = "sizeLapsed")]
        public double? SizeLapsed { get; set; }

        [JsonProperty(PropertyName = "sizeCancelled")]
        public double? SizeCancelled { get; set; }

        [JsonProperty(PropertyName = "sizeVoided")]
        public double? SizeVoided { get; set; }

        public override string ToString()
        {
            return new StringBuilder().AppendFormat("BetId={0}", BetId)

                        .AppendFormat(" : OrderType={0}", OrderType)
                        .AppendFormat(" : OrderStatus={0}", Status)
                        .AppendFormat(" : PersistenceType={0}", PersistenceType)
                        .AppendFormat(" : Side={0}", Side)
                        .AppendFormat(" : Size@Price={0}@{1}", SizeRemaining, Price)	// instead of simply Size
                        .AppendFormat(" : BspLiability={0}", BspLiability)

                        .AppendFormat(" : PlacedDate={0}", PlacedDate)
                        .AppendFormat(" : AvgPriceMatched={0}", AvgPriceMatched)

                        .AppendFormat(" : SizeMatched={0}", SizeMatched)
                        .AppendFormat(" : SizeRemaining={0}", SizeRemaining)
                        .AppendFormat(" : SizeLapsed={0}", SizeLapsed)
                        .AppendFormat(" : SizeCancelled={0}", SizeCancelled)
                        .AppendFormat(" : SizeVoided={0}", SizeVoided)

                        .ToString();
        }
    }
}

File: android/src/com/betfair/api/ng/objects/Order.java
package com.betfair.api.ng.objects;

import java.util.Date;

import org.apache.commons.lang3.builder.EqualsBuilder;
import org.apache.commons.lang3.builder.HashCodeBuilder;

public class Order {

	private String betId;
    private String orderType;
    private String status;
    private String persistenceType;
    private String side;
    private Double price;
    private Double size;
    private Double bspLiability;
    private Date placedDate;
    private Double avgPriceMatched;
    private Double sizeMatched;
    private Double sizeRemaining;
    private Double sizeLapsed;
    private Double sizeCancelled;
    private Double sizeVoided;
    
    
    public String getBetId() {
		return betId;
	}

	public void setBetId(String betId) {
		this.betId = betId;
	}

	public String getOrderType() {
		return orderType;
	}

	public void setOrderType(String orderType) {
		this.orderType = orderType;
	}

	public String getStatus() {
		return status;
	}

	public void setStatus(String status) {
		this.status = status;
	}

	public String getPersistenceType() {
		return persistenceType;
	}

	public void setPersistenceType(String persistenceType) {
		this.persistenceType = persistenceType;
	}

	public String getSide() {
		return side;
	}

	public void setSide(String side) {
		this.side = side;
	}

	public Double getPrice() {
		return price;
	}

	public void setPrice(Double price) {
		this.price = price;
	}

	public Double getSize() {
		return size;
	}

	public void setSize(Double size) {
		this.size = size;
	}

	public Double getBspLiability() {
		return bspLiability;
	}

	public void setBspLiability(Double bspLiability) {
		this.bspLiability = bspLiability;
	}

	public Date getPlacedDate() {
		return placedDate;
	}

	public void setPlacedDate(Date placedDate) {
		this.placedDate = placedDate;
	}

	public Double getAvgPriceMatched() {
		return avgPriceMatched;
	}

	public void setAvgPriceMatched(Double avgPriceMatched) {
		this.avgPriceMatched = avgPriceMatched;
	}

	public Double getSizeMatched() {
		return sizeMatched;
	}

	public void setSizeMatched(Double sizeMatched) {
		this.sizeMatched = sizeMatched;
	}

	public Double getSizeRemaining() {
		return sizeRemaining;
	}

	public void setSizeRemaining(Double sizeRemaining) {
		this.sizeRemaining = sizeRemaining;
	}

	public Double getSizeLapsed() {
		return sizeLapsed;
	}

	public void setSizeLapsed(Double sizeLapsed) {
		this.sizeLapsed = sizeLapsed;
	}

	public Double getSizeCancelled() {
		return sizeCancelled;
	}

	public void setSizeCancelled(Double sizeCancelled) {
		this.sizeCancelled = sizeCancelled;
	}

	public Double getSizeVoided() {
		return sizeVoided;
	}

	public void setSizeVoided(Double sizeVoided) {
		this.sizeVoided = sizeVoided;
	}

	public String toString() {
    	return "{"+""+"betId="+getBetId()+","+"orderType="+getOrderType()+","+"status="+getStatus()+","+"persistenceType="+getPersistenceType()+","+"side="+getSide()+","+"price="+getPrice()+","+"size="+getSize()+","+"bspLiability="+getBspLiability()+","+"placedDate="+getPlacedDate()+","+"avgPriceMatched="+getAvgPriceMatched()+","+"sizeMatched="+getSizeMatched()+","+"sizeRemaining="+getSizeRemaining()+","+"sizeLapsed="+getSizeLapsed()+","+"sizeCancelled="+getSizeCancelled()+","+"sizeVoided="+getSizeVoided()+","+"}";
    }

    public boolean equals(Object o) {
        if (!(o instanceof Order)) {
            return false;
        }

        if (this == o) {
            return true;
        }
        Order another = (Order)o;

        return new EqualsBuilder()
            .append(betId, another.betId)
            .append(orderType, another.orderType)
            .append(status, another.status)
            .append(persistenceType, another.persistenceType)
            .append(side, another.side)
            .append(price, another.price)
            .append(size, another.size)
            .append(bspLiability, another.bspLiability)
            .append(placedDate, another.placedDate)
            .append(avgPriceMatched, another.avgPriceMatched)
            .append(sizeMatched, another.sizeMatched)
            .append(sizeRemaining, another.sizeRemaining)
            .append(sizeLapsed, another.sizeLapsed)
            .append(sizeCancelled, another.sizeCancelled)
            .append(sizeVoided, another.sizeVoided)
            .isEquals();
    }

    public int hashCode() {
        return new HashCodeBuilder()
            .append(betId)
            .append(orderType)
            .append(status)
            .append(persistenceType)
            .append(side)
            .append(price)
            .append(size)
            .append(bspLiability)
            .append(placedDate)
            .append(avgPriceMatched)
            .append(sizeMatched)
            .append(sizeRemaining)
            .append(sizeLapsed)
            .append(sizeCancelled)
            .append(sizeVoided)
            .toHashCode();
    }
}
